client/src/components/TaskOptionsModal.jsx

```diff
@@ -40,46 +51,37 @@ var TaskOptionsModal = (props) => {
     }
   }
 
+//   <div style={{display: 'flex', flexDirection: 'row', gap: '5%'}}>
+//   <ContentEditable variant="body1" contentEditable={isEditing}
+//   onChange={(e)=>handleContentEditable(e, 'title')} html={task.title}
+//   />
+//   {isMobile && addToCal}
+// </div>
+// <ContentEditable variant="body1" contentEditable={isEditing}
```

Comment:
Big block comment

Reply:
The Modal I pushed is a WIP and is kind of a construction zone at the moment

Reply:
lol. Thanks for pushing it anyway--I think it will help.